cfme/tests/services/test_dialog_element_in_catalog.py

```diff
@@ -88,6 +87,18 @@ def custom_categories(appliance):
     category.delete_if_exists()
 
 
+def create_dialog(appliance, element_data):
```

Comment:
is it possible to use fallowing fixture?
https://github.com/ManageIQ/integration_tests/pull/9995/files#diff-faeac2d3d489ac01e0ebae6983c9c3cbL38

Reply:
Hello @digitronik , I have tried with fixture which you mentioned here but this fixture is test case-specific fixture so can not use as normal generic fixture. So now instead of using normal function I have created one generic fixture and automated testcase. from now new testcase also can use new implemented `create_service_dialog` fixture.